lib/helpers/add-one.js

```diff
@@ -0,0 +1,42 @@
+var PouchDBErrors = require('pouchdb-errors')
```

Comment:
`pouchdb-errors` is also missing in the dependencies. See my comment from above.